yt_dlp/extractor/filmmodu.py

```diff
@@ -0,0 +1,72 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import int_or_none
+
+
+class FilmmoduIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www.)?filmmodu.org/(?P<id>[^/]+-(?:turkce-dublaj-izle|altyazili-izle))'
+    _TESTS = [{
+        'url': 'https://www.filmmodu.org/f9-altyazili-izle',
+        'md5': '26eb61d8bab018b9189ded692c352a53',
+        'info_dict': {
+            'id': '10804',
+            'ext': 'm3u8',
+            'title': 'F9',
+            'description': 'md5:2713f584a4d65afa2611e2948d0b953c',
+            'subtitles': {
+                'tr': [{
+                    'ext': 'vtt',
+                }],
+            },
+            'thumbnail': r're:https://s[0-9]+.filmmodu.org/uploads/movie/cover/10804/xXHZeb1yhJvnSHPzZDqee0zfMb6.jpg',
+        },
+    }, {
+        'url': 'https://www.filmmodu.org/the-godfather-turkce-dublaj-izle',
+        'md5': '04c16707a945130246876b9b58b3a365',
+        'info_dict': {
+            'id': '3646',
+            'ext': 'm3u8',
+            'title': 'Baba',
+            'description': 'md5:d43fd651937cd75cc650883ebd8d8461',
+            'thumbnail': r're:https://s[0-9]+.filmmodu.org/uploads/movie/cover/3646/6xKCYgH16UuwEGAyroLU6p8HLIn.jpg',
+        },
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        title = self._og_search_title(webpage, fatal=True)
+        description = self._og_search_description(webpage)
+        thumbnail = self._og_search_thumbnail(webpage)
+        real_video_id = self._search_regex(r'var\s*videoId\s*=\s*\'([0-9]+)\'', webpage, 'video_id')
+        video_type = self._search_regex(r'var\s*videoType\s*=\s*\'([a-z]+)\'', webpage, 'video_type')
+        data = self._download_json('https://www.filmmodu.org/get-source', real_video_id, query={
+            'movie_id': real_video_id,
+            'type': video_type,
+        })
+        formats = [{
+            'url': source['src'],
+            'format_id': source.get('label'),
+            'width': int_or_none(source.get('res')),
```

Comment:
are you sure this is width and not height?

Reply:
yes it's height, put width by mistake